manifests/namespace-install.yaml

```diff
@@ -1,291 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why is this file gone?

Reply:
I honestly just ran `make codegen` and the result contained this change. I'm not sure why it is that.

Reply:
All right, can you just restore this file, and then all the builds should be in good shape. I'll figure out what's wrong there.

BTW, thanks for raising this PR!

Reply:
Done 😃

Reply:
`apiVersion: apiextensions.k8s.io/v1beta1` (line 1) is missing.

Reply:
sorry about that! It's getting late in the evening and I'm getting tired 😄  now it should be fine